phylogenetic/defaults/auspice_config.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "Real-time tracking of rabies full genome virus evolution",
+  "maintainers": [
+    {"name": "the Nextstrain team", "url": "https://nextstrain.org/team"}
+  ],
```

Comment:
```suggestion
  "maintainers": [
    {"name": "Kim Andrews", "url": "https://bedford.io/team/kim-andrews/"},
    {"name": "the Nextstrain team", "url": "https://nextstrain.org/team"}
  ],
```

Reply:
Done in [c49d1c7](https://github.com/nextstrain/rabies/pull/3/commits/c49d1c768bb0c321ca9a2ba5b90fa79ac9834c74)